README.md

```diff
@@ -1,60 +1,58 @@
 # libspdm is a sample implementation that follows the DMTF [SPDM](https://www.dmtf.org/standards/pmci) specification
 
-## Feature
+## Features
 
-1) Specification
+1) Specifications
 
-   The SPDM and secured message follow :
+   The SPDM and secured message libraries follow :
 
    [DSP0274](https://www.dmtf.org/dsp/DSP0274)  Security Protocol and Data Model (SPDM) Specification (version [1.0.1](https://www.dmtf.org/sites/default/files/standards/documents/DSP0274_1.0.1.pdf), version [1.1.1](https://www.dmtf.org/sites/default/files/standards/documents/DSP0274_1.1.1.pdf) and version [1.2.0](https://www.dmtf.org/sites/default/files/standards/documents/DSP0274_1.2.0.pdf))
 
    [DSP0277](https://www.dmtf.org/dsp/DSP0277)  Secured Messages using SPDM Specification (version [1.0.0](https://www.dmtf.org/sites/default/files/standards/documents/DSP0277_1.0.0.pdf))
 
-   The MCTP and secured MCTP follow :
+   MCTP and secured MCTP follow :
 
    [DSP0275](https://www.dmtf.org/dsp/DSP0275)  Security Protocol and Data Model (SPDM) over MCTP Binding Specification (version [1.0.0](https://www.dmtf.org/sites/default/files/standards/documents/DSP0275_1.0.0.pdf))
 
    [DSP0276](https://www.dmtf.org/dsp/DSP0276)  Secured MCTP Messages over MCTP Binding Specification (version [1.0.0](https://www.dmtf.org/sites/default/files/standards/documents/DSP0276_1.0.0.pdf))
 
-   The PCI DOE / IDE follow :
+   PCI DOE / IDE follow :

```

Comment:
@jyao1 can we remove CMA and IDE? libspdm doesn't appear to do anything with either of those specifications.

Reply:
Agree.

I think we should remove all PCI related spec and just use [PCI Express Base Specification Revision 6.0, Version 1.0](https://members.pcisig.com/wg/PCI-SIG/document/16609)